scripts/scil_screenshot_bundle.py

```diff
@@ -76,43 +81,79 @@ def prepare_data_for_actors(bundle_filename, reference_filename,
     reference_data = reference_img.get_data()
     reference_affine = reference_img.get_affine()
 
-    target_template_img = nib.load(target_template_filename)
-    target_template_data = target_template_img.get_data()
-    target_template_affine = target_template_img.affine
+    if target_template_filename:
+        target_template_img = nib.load(target_template_filename)
+        target_template_data = target_template_img.get_data()
+        target_template_affine = target_template_img.affine
+
+        # Register the DWI data to the template
+        transformed_reference, transformation = register_image(target_template_data,
+                                                               target_template_affine,
+                                                               reference_data,
+                                                               reference_affine)
+
+        streamlines = transform_streamlines(streamlines,
+                                            np.linalg.inv(transformation))
```

Comment:
This inversion could be optimized since it is either an Affine or a Rigid transformation